docs/source/using-executorch-export.md

```diff
@@ -1,3 +1,173 @@
+
+```python
+import torch
+from executorch.backends.xnnpack.partition.xnnpack_partitioner import XnnpackPartitioner
+from executorch.exir import to_edge_transform_and_lower
+from torch.export import Dim, export
+
+class Model(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.seq = torch.nn.Sequential(
+            torch.nn.Conv2d(1, 8, 3),
+            torch.nn.ReLU(),
+            torch.nn.Conv2d(8, 16, 3),
+            torch.nn.ReLU(),
+            torch.nn.AdaptiveAvgPool2d([1,1])
+        )
+        self.linear = torch.nn.Linear(16, 10)
+    
+    def forward(self, x):
+        y = self.seq(x)
+        y = torch.flatten(y, 1)
+        y = self.linear(y)
+        return y
+
+model = Model()
+inputs = (torch.randn(1,1,16,16),)
+dynamic_shapes = {
+    "x": {
+        2: Dim("h", min=16, max=1024),
+        3: Dim("w", min=16, max=1024),
+    }
+}
+
+exported_program = export(model, inputs, dynamic_shapes=dynamic_shapes)
+executorch_program = to_edge_transform_and_lower(
+    exported_program,
+    partitioner = [XnnpackPartitioner()]
+).to_executorch()
+
+with open("model.pte", "wb") as file:
+    file.write(executorch_program.buffer)
+```
+
+This yields a `model.pte` file which can be run on mobile devices.
+
+## Testing the Model
+
+Before integrating the runtime code, it is common to test the exported model from Python. This can be used to evaluate model accuracy and sanity check behavior before moving to the target device. Note that not all hardware backends are available from Python, as they may require specialized hardware to function. See the specific backend documentation for more information on hardware requirements and the availablilty of simulators. The XNNPACK delegate used in this example is always available on host machines.
+
+```python
+from executorch.runtime import Runtime
+
+runtime = Runtime.get()
+
+input_tensor = torch.randn(1, 3, 32, 32)
+program = runtime.load_program("model.pte")
+method = program.load_method("forward")
+outputs = method.execute([input_tensor])
+```
+
+For more information, see [Runtime API Reference](executorch-runtime-api-reference.md).
+
+## Next Steps
+
+The PyTorch and ExecuTorch export and lowering APIs provide a high level of customizability to meet the needs of diverse hardware and models. See [torch.export](https://pytorch.org/docs/main/export.html) and [Export API Reference](export-to-executorch-api-reference.md) for more information.
+
+For advanced use cases, see the following:
+- [Memory Planning](compiler-memory-planning.md) for information on controlling memory placement and planning.
+- [Backends and Delegates](compiler-delegate-and-partitioner.md) for information on how backends interact with the lowering process.
```

Comment:
`compiler-delegate-and-partitioner.md` page is really not for users but more for backend authors. 

i don't really see a reason why a user would want to read that as next steps.

Reply:
ompiler-custom-compiler-passes.html is probably appropriate for end users